resources/views/masters/_masters.blade.php

```diff
@@ -15,7 +15,7 @@
     <tbody>
       <?php $index = 0; ?>
       @foreach ($masters as $master)
-      <tr role="button" class="@if ($marks->contains($master->id)) marked @endif" data-id="{{ $master->id }}" data-index="{{ $start + $index }}">
+      <tr role="button" class="@if ($marks->contains($master->id)) marked @endif" data-id="{{ $master->id }}" data-index="{{ $start + $index }}" data-sort-column="{{$sortColumn}}" data-sort-direction="{{$sortDirection}}">
```

Comment:
Does the `data-sort-direction` value need a similar callNumber check as above?

Reply:
no, because this ternary flips the sort direction in anticipation of the column heading being clicked again. here it's just recording what the present sort is, to send to the table selection resolveRange query